full_snapshot.proto

```diff
@@ -497,6 +499,7 @@ message RelationStatistic {
   NullTimestamp last_autovacuum = 46;  // Last time at which this table was vacuumed by the autovacuum daemon
   NullTimestamp last_analyze = 47;     // Last time at which this table was manually analyzed
   NullTimestamp last_autoanalyze = 48; // Last time at which this table was analyzed by the autovacuum daemon
+  int32 toast_reltuples = 49;          // Number of live rows in the TOAST table. -1 indicating that the row count is unknown
```

Comment:
I wonder if we shouldn't also be using float here, like for the regular `reltuples`? The reason this is not a precise measurement is that Postgres doesn't always actually count the tuples on all pages, but rather estimates them (e.g. after running a vacuum).

Reply:
As far as I can tell `reltuples` is always an integer in practice. How would it sometimes end up as a float?

Reply:
Thread here, though I'll have to re-read properly understand it https://postgrespro.com/list/thread-id/1614053

Reply:
Hmm. That thread is quite ancient, so not sure if it captured all the aspects on this, specifically to how vacuum estimates reltuples.

If we were to use an integer here, it would have to be 64-bit though (per the thread you linked). I was mostly thinking we stay consistent here with Postgres itself, and use a float32 (thus that's how we did it for the regular reltuples).

Reply:
Okay, updated to use float32